airflow/jobs/scheduler_job.py

```diff
@@ -141,8 +140,8 @@ def _run_file_processor(result_channel,
 
         try:
             # redirect stdout/stderr to log
-            with redirect_stdout(StreamLogWriter(log, logging.INFO)),\
-                    redirect_stderr(StreamLogWriter(log, logging.WARN)):
+            with StdoutToLog(log, logging.INFO), \
+                    StderrToLog(log, logging.WARN):
```

Comment:
Is there any need to make this change? What does this achieve?

Reply:
Just made this change for consistency, but you are correct that it is not necessary. Probably not actually helpful either on second look.